src/mesh/mesh_refinement.hpp

```diff
@@ -76,6 +76,16 @@ class MeshRefinement {
   // and/or in BoundaryValues::ProlongateBoundaries() (for SMR and AMR)
   int AddToRefinement(ParArrayND<Real> pvar_cc, ParArrayND<Real> pcoarse_cc);
   int AddToRefinement(FaceField *pvar_fc, FaceField *pcoarse_fc);
+  std::vector<int> GetInternals() {
```

Comment:
Given that `int32_t` is used above to read, I suggest to also be more specific here.

Reply:
Will do.